ethereumj-core/src/main/java/org/ethereum/net/rlpx/MessageCodesResolver.java

```diff
@@ -34,12 +36,21 @@ public void init(List<Capability> caps) {
         int offset = P2pMessageCodes.USER.asByte() + 1;
 
         for (Capability capability : caps) {
+
             if (capability.getName().equals(Capability.ETH)) {
                 setEthOffset(offset);
                 EthVersion v = fromCode(capability.getVersion());
                 offset += EthMessageCodes.values(v).length;
             }
 
+            if (capability.getName().equals(Capability.PAR)) {
+                // TODO: Why???
+                setParOffset(33);
+                ParVersion v = ParVersion.fromCode(capability.getVersion());
+                // TODO: need we?
+                offset += ParMessageCodes.values(v).length;
```

Comment:
I think it doesn't matter here since `PAR` and (`SHH` and/or `BZZ`) are mutually exclusive